master/internal/stream/system.go

```diff
@@ -614,7 +643,7 @@ func prepareMessageWithCache(
 	if *cache != nil {
 		return *cache
 	}
-	jbytes, err := json.Marshal(obj)
+	jbytes, err := json.Marshal(obj) // maybe json.RawMessage
```

Comment:
what does this comment mean?

Reply:
I think this is a leftover comment from the rebase I did - I'll get rid of it